File: zen/Zen.php
<?php

namespace MattHarley\Zen;

class Zen
{
    /**
     * The Zen of Python:
     * use MattHarley\Zen\Zen;
     * $myZen = new Zen();
     * $myZen->importThis();
     */
    public function importThis()
    {
        $output = [];
        exec('python -c \'import this\'', $output);
        return $output;
    }

    public function beautifulIsBetterThanUgly()
    {
        $beautiful = 133 == 0133 && "133" == "0133" || 123 == "123foo" ? -1 : 0;
        $ugly = json_decode('{"hello": "world"');
        return $beautiful > $ugly;
    }

    public function errorsShouldNeverPassSilently($dividend, $divisor)
    {
        return @($dividend / $divisor);
    }
}

File: zen/zen.py
import json

class Zen:
    def beautiful_is_better_than_ugly(self):
        try:
            eval('beautiful = -1 if 133 == 0133 and "133" == "0133" or "133" == "0133" else 0')
        except SyntaxError:
            beautiful = -1

        try:
            ugly = json.loads('{"hello": "world"')
        except json.JSONDecodeError:
            ugly = None

        return beautiful > ugly

    def errors_should_never_pass_silently(dividend, divisor):
        try:
            result = dividend / divisor
        except ZeroDivisionError:
            pass

        return result
